assets/models/EditorArrow.tscn

```diff
@@ -0,0 +1,17 @@
+[gd_scene load_steps=4 format=2]
```

Comment:
This PR seems to have been mixed up with the editor arrow one.
You should do the following: create a new local branch from this (this in effect saves this branch with a new name), delete this branch locally, create this branch again from master, and then cherry-pick all the right commits from the new branch where you saved the state of this branch